### PR TITLE
OrderDetails

### PR DESCRIPTION
**Pull request by goody-1**

### CODE REVIEW

File: src/models/__init__.py [hunk 1]
```diff
@@ -3,3 +3,5 @@
 db = SQLAlchemy()
 
 from .customer import Customer
+from .orderdetails import OrderDetails
```
Comment (samdoghor) [inline src/models/__init__.py:6]: table name should be in snake-case

File: src/models/orderdetails.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+from .abc import BaseModel, MetaBaseModel
+
+
+class OrderDetails(db.Model, BaseModel, metaclass=MetaBaseModel):
+    """OrderDetails Model"""
+
+    __tablename__ = "order_details"
+
+    id = db.Column(db.Integer, primary_key=True)
+    status = db.Column(db.Boolean, nullable=False)
```
Comment (samdoghor) [inline src/models/orderdetails.py:14]: Move to foreign key, a different model will be created for status

File: src/models/orderdetails.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    quantity = db.Column(db.Integer, nullable=False)
+    sku = db.Column(db.String(50), nullable=False)
+
+    #foreign keys
```
Comment (samdoghor) [inline src/models/orderdetails.py:18]: The table names you are referencing should be in lower case

**Review by samdoghor — CHANGES_REQUESTED** [review]
Include migration files in your next PR

File: src/models/orderdetails.py (hunk 2 above)
Comment (goody-1) [reply src/models/orderdetails.py:14]: Will the status still be **boolean** type?

**Pull request closed without merging**